Source/WebKit/UIProcess/WebScreenOrientationManagerProxy.cpp

```diff
@@ -122,15 +122,15 @@ void WebScreenOrientationManagerProxy::lock(WebCore::ScreenOrientationLockType l
     if (resolvedLockedOrientation != m_currentlyLockedOrientation) {
         bool didLockOrientation = false;
 #if ENABLE(FULLSCREEN_API)
-        if (m_page.fullScreenManager() && m_page.fullScreenManager()->isFullScreen()) {
-            if (!m_page.fullScreenManager()->lockFullscreenOrientation(resolvedLockedOrientation)) {
+        if (m_page->fullScreenManager() && m_page->fullScreenManager()->isFullScreen()) {
+            if (!m_page->fullScreenManager()->lockFullscreenOrientation(resolvedLockedOrientation)) {
                 m_currentLockRequest(WebCore::Exception { WebCore::ExceptionCode::NotSupportedError, "Screen orientation locking is not supported"_s });
                 return;
             }
             didLockOrientation = true;
         }
 #endif
-        if (!didLockOrientation && !m_page.uiClient().lockScreenOrientation(m_page, resolvedLockedOrientation)) {
+        if (!didLockOrientation && !m_page->uiClient().lockScreenOrientation(m_page.get(), resolvedLockedOrientation)) {
```

Comment:
Seems we need to pass a `protectedPage()` to `lockScreenOrientation()`.

Reply:
fixed.